src/test/java/uk/gov/pay/adminusers/resources/ServiceUpdateOperationValidatorTest.java

```diff
@@ -152,21 +128,9 @@ public void shouldFail_whenUpdateRedirectToService_whenOpIsNotReplace() {
         shouldFailForAddOperation("redirect_to_service_immediately_on_terminal_state", true);
     }
 
-    @Test
-    public void shouldFail_whenUpdateRedirectToService_whenOpIsNotPresent() {
-        ObjectNode payload = mapper.createObjectNode();
-        payload.put("path", "redirect_to_service_immediately_on_terminal_state");
-        payload.put("value", true);
-
-        List<String> errors = serviceUpdateOperationValidator.validate(payload);
-
-        assertThat(errors.size(), is(1));
-        assertThat(errors, hasItem("Field [op] is required"));
-    }
-
```

Comment:
Removed this one because we already have a test checking that errors are thrown when the "op" and "value" fields are missing. This is common logic so don't need a test for each path.